cmd/sonobuoy/app/args.go

```diff
@@ -1,6 +1,8 @@
 /*
 Copyright 2018 Heptio Inc.
 
+© 2022 Nokia
```

Comment:
So the historic heptio copyrights remain but in general we now ask that changes be tagged with `Copyright <yr> the Sonobuoy Project contributors`

Reply:
The copyright message has been updated as suggested